pragma-node/src/infra/repositories/entry_repository.rs

```diff
@@ -427,13 +418,11 @@ pub async fn get_twap_prices_between(
             pair_id = $1
             AND
             bucket BETWEEN $2 AND $3
-            {}
         ORDER BY
             time DESC;
     ",
         get_interval_specifier(entry_params.interval, true)?,
         get_table_suffix(entry_params.data_type)?,
-        get_expiration_timestamp_filter(entry_params.data_type, &entry_params.expiry)?,
```

Comment:
We removed the expiration filtering? Why? It will cause problems when dealing with perp <=> future, no?